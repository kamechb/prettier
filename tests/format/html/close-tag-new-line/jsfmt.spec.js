run_spec(__dirname, ["html"]);
run_spec(__dirname, ["html"], {closeTagNewLine: true});
run_spec(__dirname, ["html"], {closeTagNewLine: true, singleAttributePerLine: true, htmlWhitespaceSensitivity: "ignore"});
run_spec(__dirname, ["html"], {closeTagNewLine: true, singleAttributePerLine: true, bracketSameLine: true, htmlWhitespaceSensitivity: "ignore"});
