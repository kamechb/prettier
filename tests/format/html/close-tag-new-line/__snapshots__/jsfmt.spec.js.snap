// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`close-tag.html - {"closeTagNewLine":true,"singleAttributePerLine":true,"bracketSameLine":true,"htmlWhitespaceSensitivity":"ignore"} format 1`] = `
====================================options=====================================
bracketSameLine: true
closeTagNewLine: true
htmlWhitespaceSensitivity: "ignore"
parsers: ["html"]
printWidth: 80
singleAttributePerLine: true
                                                                                | printWidth
=====================================input======================================
<div class="wrapper" id="main" data-a="aaaaaaaaa" data-b="bbbbbbbbb" data-c="ccccccccccccccccccc"></div>
<span id="name" class="name">Jack</span>
<p class="article" id="article">article content</p>
<app-name class="app" id="app"></app-name>
<link href="/aaaa.css" rel="stylesheet">
<br>

=====================================output=====================================
<div
  class="wrapper"
  id="main"
  data-a="aaaaaaaaa"
  data-b="bbbbbbbbb"
  data-c="ccccccccccccccccccc">
</div>
<span
  id="name"
  class="name">
  Jack
</span>
<p
  class="article"
  id="article">
  article content
</p>
<app-name
  class="app"
  id="app">
</app-name>
<link
  href="/aaaa.css"
  rel="stylesheet" />
<br />

================================================================================
`;

exports[`close-tag.html - {"closeTagNewLine":true,"singleAttributePerLine":true,"htmlWhitespaceSensitivity":"ignore"} format 1`] = `
====================================options=====================================
closeTagNewLine: true
htmlWhitespaceSensitivity: "ignore"
parsers: ["html"]
printWidth: 80
singleAttributePerLine: true
                                                                                | printWidth
=====================================input======================================
<div class="wrapper" id="main" data-a="aaaaaaaaa" data-b="bbbbbbbbb" data-c="ccccccccccccccccccc"></div>
<span id="name" class="name">Jack</span>
<p class="article" id="article">article content</p>
<app-name class="app" id="app"></app-name>
<link href="/aaaa.css" rel="stylesheet">
<br>

=====================================output=====================================
<div
  class="wrapper"
  id="main"
  data-a="aaaaaaaaa"
  data-b="bbbbbbbbb"
  data-c="ccccccccccccccccccc"
>
</div>
<span
  id="name"
  class="name"
>
  Jack
</span>
<p
  class="article"
  id="article"
>
  article content
</p>
<app-name
  class="app"
  id="app"
>
</app-name>
<link
  href="/aaaa.css"
  rel="stylesheet"
/>
<br />

================================================================================
`;

exports[`close-tag.html - {"closeTagNewLine":true} format 1`] = `
====================================options=====================================
closeTagNewLine: true
parsers: ["html"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<div class="wrapper" id="main" data-a="aaaaaaaaa" data-b="bbbbbbbbb" data-c="ccccccccccccccccccc"></div>
<span id="name" class="name">Jack</span>
<p class="article" id="article">article content</p>
<app-name class="app" id="app"></app-name>
<link href="/aaaa.css" rel="stylesheet">
<br>

=====================================output=====================================
<div
  class="wrapper"
  id="main"
  data-a="aaaaaaaaa"
  data-b="bbbbbbbbb"
  data-c="ccccccccccccccccccc"
>
</div>
<span id="name" class="name">Jack</span>
<p class="article" id="article">article content</p>
<app-name class="app" id="app"></app-name>
<link href="/aaaa.css" rel="stylesheet" />
<br />

================================================================================
`;

exports[`close-tag.html format 1`] = `
====================================options=====================================
parsers: ["html"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<div class="wrapper" id="main" data-a="aaaaaaaaa" data-b="bbbbbbbbb" data-c="ccccccccccccccccccc"></div>
<span id="name" class="name">Jack</span>
<p class="article" id="article">article content</p>
<app-name class="app" id="app"></app-name>
<link href="/aaaa.css" rel="stylesheet">
<br>

=====================================output=====================================
<div
  class="wrapper"
  id="main"
  data-a="aaaaaaaaa"
  data-b="bbbbbbbbb"
  data-c="ccccccccccccccccccc"
></div>
<span id="name" class="name">Jack</span>
<p class="article" id="article">article content</p>
<app-name class="app" id="app"></app-name>
<link href="/aaaa.css" rel="stylesheet" />
<br />

================================================================================
`;
